### PR TITLE
add option to manage root_dir

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -3,10 +3,11 @@
 
   case $::tftp::daemon {
     default: {
-      file { $::tftp::root:
-        ensure => directory,
+      if $tftp::manage_root_dir {
+        file { $::tftp::root:
```
Comment (ekohl) [inline manifests/config.pp]: Should this also use ```ensure_resource``` just to be sure and reduce the likelihood to start with?

File: manifests/config.pp [hunk 2]
```diff
@@ -3,10 +3,9 @@
 
   case $::tftp::daemon {
     default: {
-      file { $::tftp::root:
-        ensure => directory,
+      if $tftp::manage_root_dir {
+        ensure_resource('file', $::tftp::root, {'ensure' => 'directory'})
       }
-
```
Comment (ekohl) [inline manifests/config.pp]: Being nit picky: I like this newline :)

Comment (bastelfreak) [reply manifests/config.pp]: readded

**Comment by mmoll** [comment]
merged, danke @bastelfreak!

**Pull request merged**